imap_processing/hi/hi_cdf_attrs.py

```diff
@@ -0,0 +1,143 @@
+"""Shared attribute values for IMAP-Hi CDF files."""
+from imap_processing.cdf.defaults import GlobalConstants
+from imap_processing.cdf.global_attrs import (
+    AttrBase,
+    GlobalDataLevelAttrs,
+    GlobalInstrumentAttrs,
+)
+from imap_processing.hi import __version__
+
+text = (
+    "IMAP-Hi consists of two identical, single-pixel"
+    "high-energy energetic neutral atom (ENA) imagers"
+    "mounted at fixed angles of 90 and 45 degrees"
+    "relative to the spacecraft spin axis. These "
+    "imagers measure neutral atoms entering our solar "
+    "system from the outer edge of the heliosphere as "
+    "they move towards the Sun. The ENA imagers collect"
+    " the neutral atoms, sort them by type, and then map "
+    "their incident direction from the outer heliosphere. "
+    "IMAP-Hi uses a time-of-flight (TOF) section to "
+    "identify hydrogen (H) and helium (He) and heavier "
+    "atoms such as carbon (C), nitrogen (N), oxygen (O), "
+    "and neon (Ne). With each spin of the spacecraft, the "
+    "imagers sample swaths in the sky that include the "
+    "ecliptic poles and four additional locations in the "
+    "ecliptic plane. Some low latitude regions, that "
+    "contain ENA emissions from the nose and tail of the "
+    "heliosphere, as well as most of the IBEX Ribbon and "
+    "Belt, are sampled twice within as little as 1.5 "
+    "months which allows it to explore short ENA "
+    "variability."
+    "IMAP-Hi's design and assembly is led by Los Alamos "
+    "National Laboratory (LANL) in collaboration with "
+    "Southwest Research Institute (SwRI), University of "
+    "New Hampshire (UNH), and University of Bern (UBe)."
+    "See https://imap.princeton.edu/instruments/mag for "
+    "more details."
+)
+
+hi_base = GlobalInstrumentAttrs(
+    __version__,
+    "IMAP-Hi>IMAP High-Energy Energetic Neutral Atom Imager",
+    text,
+    "Particles (space)",
+)
+
+# Direct event attrs
+met_subseconds_attrs = AttrBase(
```

Comment:
Would it make sense to combine seconds and subseconds into a single attribute?

Reply:
they are different values and has different description. That's why I kept them separate. Can you elaborate on what you are thinking?